README.md

```diff
@@ -93,6 +93,12 @@ file:path/to/yourproject/file          read directly from a path on disk (any si
 registry:yourrepo/yourimage:tag        pull image directly from a registry (no container runtime required)
 ```
 
+Syft can exclude files and paths from being scanned within a source by using
+one or more `--exclude` parameters:
+```
+syft packages path/to/dir --exclude **/*.png --exclude **/generated/**
+```
+
```

Comment:
It'd be nice to have a section header for this content. A lot of folks have been asking about this feature, so being able to give them a direct link to this section would be really valuable!